srcpkgs/glirc/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'glirc'
+pkgname=glirc
+version=2.38
+revision=1
+wrksrc="irc-core-${version}"
+hostmakedepends="cabal-install"
+makedepends="ncurses-devel openssl-devel"
+short_desc="Console IRC client written in Haskell"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="ISC"
+homepage="https://github.com/glguy/irc-core"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=abb51831a3748c22eb5f8fbeee1fa71d2a99a6f92331976eef980f8f7d0c9680
+nocross="cabal does not support cross compilation"
+nopie=yes	# ghc is currently built without PIE support
+
+_cabal_opts="-g"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	_cabal_opts+=" --ghc-option -latomic"
+fi
+
+if [ "$XBPS_MAKEJOBS" ]; then
+	_cabal_opts+=" --jobs=$XBPS_MAKEJOBS"
+fi
+
+_ghcver="$(xbps-query -S ghc --prop pkgver)"
```

Comment:
There should be no external commands run on top level of template. Please either remove this completely and use wildcard, or move inside of do_build replacing with `ghc --version | sed _something_`.

Reply:
> There should be no external commands run on top level of template.

okay, I've removed them in the latest commit.  
But may I know the reason behind this rule? Also, is it mentioned in the [Manual.md](https://github.com/void-linux/void-packages/blob/master/Manual.md)?

Reply:
Answering myself: One of the reasons will be that the template file is sourced many times, thus any command run on the top level will cause significant slowdowns.